source-container-build/app/source_build.py

```diff
@@ -445,6 +445,19 @@ def resolve_source_image_by_manifest(image: str) -> str | None:
         log.info("Source container image %s does not exist.", source_image)
 
 
+def resolve_source_image(binary_image: str, registries_allow_list: list[str]) -> str | None:
+    allowed = urlparse("docker://" + binary_image).netloc in registries_allow_list
+    if allowed:
+        return resolve_source_image_by_version_release(binary_image)
+    else:
+        logger.info(
+            "Image %s does not come from supported allowed registry. "
+            "Skip handling the sources for it.",
+            binary_image,
+        )
```

Comment:
The log message seems a bit misleading - the script says "Skip handling the source" and then immediately tries to resolve the source image by digest

Reply:
Good catch :+1: PTAL